extensions-contrib/azure-extensions/src/main/java/org/apache/druid/storage/azure/AzureStorage.java

```diff
@@ -33,10 +36,15 @@
 import java.io.InputStream;
 import java.net.URISyntaxException;
 import java.util.ArrayList;
+import java.util.EnumSet;
 import java.util.List;
 
+/**
+ * Abstracts the Azure storage layer. Makes direct calls to Azure file system.
+ */
 public class AzureStorage
 {
+  private static final boolean USE_FLAT_BLOB_LISTING = true;
 
   private final Logger log = new Logger(AzureStorage.class);
```

Comment:
Please make this a static final variable.

Reply:
Will fix this as part of a subsequent change to this area of the code.